dandiapi/api/models/garbage_collection.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from django.db import models
+
+
+class GarbageCollectionEvent(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    type = models.CharField(
+        max_length=255, help_text='The model name of the records that were garbage collected.'
+    )
+    records = models.JSONField(
```

Comment:
Is there a particular reason you chose to store a collection of records in one "Event" object, rather than one row per object? I.e. `records` would become `record`, and `garbage_collection_event_id` could be a foreign key to a separate table that stores the events.

One thing that it seems would be easier in that model is searching for a particular garbage collected upload or asset blob.

Reply:
(I realize this is how it's outlined in the design doc, but this question is just coming to me now.)